api/helpers/game-states/create-socket-event.js

```diff
@@ -14,19 +15,29 @@ module.exports = {
     gameState: {
       type: 'ref',
       description: 'Game state object',
-      required: true,
     },
   },
 
   fn: async function ({ game, gameState }, exits) {
     try {
+      if (!game.status === GameStatus.CREATED && !gameState) {
+        throw new Error('GameState Required to create socket event');
+      }
       // Combine game and gamestate users and delete passwords
-      const p0 = { ...game.p0, ...gameState.p0 };
+      const p0 = { ...game.p0, ...gameState?.p0 };
       delete p0.encryptedPassword;
-      const p1 = { ...game.p1, ...gameState.p1 };
+      const p1 = { ...game.p1, ...gameState?.p1 };
       delete p1.encryptedPassword;
       const players = [ p0, p1 ];
 
+      // Early return if not in game yet
+      if (!gameState) {
```

Comment:
Early return here, since the game table has almost everything the client needs for the lobby, adding players to add the players to the game store.


wondering if we should make a new variable called something like `gameCreated` and use that in the spot above and here